index.html

```diff
@@ -159,91 +158,35 @@ <h2 class="profession">Software Developer</h2>
             <section class="works">
                 <h2 id="work-head">MY RECENT WORK</h2>
                 <div class="wrapper">
-                    <div class="item">
-                        <div class="fb fb1">
-                            <img src="Img Placeholder.png" alt="Project 1">
-                            <div class="font-change">
-                                <h3>Multi-Post Stories</h3>
-                                <p class="description">A daily selection of privately personalized reads; no accounts or
-                                    sign-ups required. has been the
-                                    industry's standard dummy text ever since the 1500s, when an unknown printer took a
-                                    standard
-                                    dummy
-                                    text.
-                                </p>
-                                <ul class="work-lang">
-                                    <li>html</li>
-                                    <li>css</li>
-                                    <li>bootstrap</li>
-                                    <li>ruby</li>
-                                </ul>
-                                <button type="button">See Project</button>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="item">
-                        <div class="fb fb2">
-                            <img src="Img Placeholder.png" alt="Project">
-                            <div class="font-change">
-                                <h3>Multi-Post Stories</h3>
-                                <p class="description">A daily selection of privately personalized reads; no accounts or
-                                    sign-ups required. has been the
-                                    industry's standard dummy text ever since the 1500s, when an unknown printer took a
-                                    standard
-                                    dummy
-                                    text.
-                                </p>
-                                <ul class="work-lang">
-                                    <li>html</li>
-                                    <li>css</li>
-                                    <li>bootstrap</li>
-                                    <li>ruby</li>
-                                </ul>
-                                <button type="button">See Project</button>
+                </div>
+                <div class="pop-container">
+                    <div class="pop-overlay">
+                        <div class="pop-content">
+                            <div class="pop-header">
```

Comment:
- [OPTIONAL] It would be better if your fix the background of your pop-overlay with
`background-color:  #100f45;`. It will allow that users watched your project well without anything which affects the visualization.

![modal-picture](https://user-images.githubusercontent.com/79876959/141352688-7e5658df-a2d0-46e7-a350-064cb14d70ba.PNG)

![modal-picture-2](https://user-images.githubusercontent.com/79876959/141352743-674e7ccd-facd-4040-a611-71a656720486.PNG)